### PR TITLE
Keeton

### PR DESCRIPTION
**Pull request by KeetonMartin**
My branch contains my work on the "View All Scholarships" functionality, and Will2's work on "Delete". Maybe it also contains work on edit/create from Iryna? Not totally clear to me. 

I also modified the create html page and the View All page to use bootstrap, and finally I fixed some bugs that I created by merging Will's work.

Screen Shot 2022-04-04 at 7 41 28 PM

Screen Shot 2022-04-04 at 7 41 48 PM

See View All and Create pages in these screenshots.

### CODE REVIEW

**Comment by qubist** [comment]
I get a crash when deleting the last remaining scholarship in the list:

### To recreate:
1. Create a few (n) scholarships
2. Delete the first n-1. This should work fine.
3. Delete the last one. This will crash express with this error again:

```
Trying to delete scholarship: 624ca145768aee859a5cdeb5
Deleted scholarship with id 624ca145768aee859a5cdeb5
node:events:353
      throw er; // Unhandled 'error' event
      ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:278:15)
    at ServerResponse.setHeader (node:_http_outgoing:563:11)
    at ServerResponse.header (/Users/will/Documents/Haverford/-CMSC-353-Software-Engineering/swe-project-team-3/scholarshipr/node_modules/express/lib/response.js:776:10)
    at ServerResponse.contentType (/Users/will/Documents/Haverford/-CMSC-353-Software-Engineering/swe-project-team-3/scholarshipr/node_modules/express/lib/response.js:604:15)
    at /Users/will/Documents/Haverford/-CMSC-353-Software-Engineering/swe-project-team-3/scholarshipr/index.js:111:9
    at /Users/will/Documents/Haverford/-CMSC-353-Software-Engineering/swe-project-team-3/scholarshipr/node_modules/mongoose/lib/model.js:4998:18
    at processTicksAndRejections (node:internal/process/task_queues:75:11)
Emitted 'error' event on Function instance at:
    at /Users/will/Documents/Haverford/-CMSC-353-Software-Engineering/swe-project-team-3/scholarshipr/node_modules/mongoose/lib/model.js:5000:15
    at processTicksAndRejections (node:internal/process/task_queues:75:11) {
  code: 'ERR_HTTP_HEADERS_SENT'
}
[nodemon] app crashed - waiting for file changes before starting...
```

Same thing that happened in my branch, but it's being a problem now.

I'm sending an email about it to Chris and Al.

**Comment by qubist** [comment]
I'm good to merge this now even with that bug, so we have more common ground to work from adding new features

**Review by qubist — APPROVED** [review]

File: scholarshipr/Scholarship.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ var scholarshipSchema = new Schema({
     description: {type: String, required: false, unique: false},
     dollarAmount: {type: Number, required: false, unique: false},
     // gender: {type: Enumerator, required: false, unique: false},
-    approvalStatus: {type: Boolean, required: true, unique: false},
+    approvalStatus: {type: String, required: true, unique: false},
```
Comment (ikhovryak) [inline scholarshipr/Scholarship.js:15]: I think we should keep _approvalStatus_ as a boolean for future suggested scholarship features and usability by HTML toggles.

Comment (KeetonMartin) [reply scholarshipr/Scholarship.js:15]: On line 32 of index.js we have some code that handles this I think. I'm not totally sure which way of handling it is best but it seems to currently be working to me.

**Review by ikhovryak — APPROVED** [review]

**Review by Williamlaw718 — APPROVED** [review]

**Pull request merged**